src/effects/effectsmanager.cpp

```diff
@@ -218,15 +218,15 @@ EffectRackPointer EffectsManager::getEffectRack(const QString& group) {
     return m_pEffectChainManager->getEffectRack(group);
 }
 
-void EffectsManager::setupDefaults() {
-    //m_pEffectChainManager->loadEffectChains();
-
+void EffectsManager::setup() {
     // Add a general purpose rack
+    QList<std::pair<EffectChainPointer, QDomElement>> savedChains;
+    savedChains = m_pEffectChainManager->loadEffectChains();
+
     StandardEffectRackPointer pStandardRack = addStandardEffectRack();
-    pStandardRack->addEffectChainSlot();
-    pStandardRack->addEffectChainSlot();
-    pStandardRack->addEffectChainSlot();
-    pStandardRack->addEffectChainSlot();
```

Comment:
I think the regression is caused by the missing empty Slots in case of no stored effect are available.